docs/documentation/_data/deckhouse-alerts.yml

```
@@ -3437,6 +3437,21 @@ alerts:
         Alerting dead man's switch.
       severity: "4"
       markupFormat: default
+    - name: DeckhouseEditionNotFound
+      sourceFile: modules/340-monitoring-deckhouse/monitoring/prometheus-rules/deckhouse.yaml
+      moduleUrl: 340-monitoring-deckhouse
+      module: monitoring-deckhouse
+      edition: ce
+      description: |
+        Please specify Deckhouse Kubernetes Platform edition in `{{ $labels.module }}` ModuleConfig.
+        ```bash
+        d8 k patch moduleconfig {{ $labels.module }} --type merge --patch '{"spec": {"settings": {"license": {"edition": "DKP_EDITION"}}}}'
+        ```
+        Where `DKP_EDITION` is the edition you want to use like `CE`, `EE`, etc.
```

```suggestion
        Specify a Deckhouse Kubernetes Platform edition in `{{ $labels.module }}` ModuleConfig.

        ```bash
        d8 k patch moduleconfig {{ $labels.module }} --type merge --patch '{"spec": {"settings": {"license": {"edition": "DKP_EDITION"}}}}'
        ```

        Where `DKP_EDITION` is the edition you want to use: `CE`, `BE`, `EE`, `SE`, or `SE-plus`.
```